src/backend/main.ts

```diff
@@ -1065,8 +1070,16 @@ ipcMain.handle(
       sessionPlaytime + tsStore.get(`${appName}.totalPlayed`, 0)
     tsStore.set(`${appName}.totalPlayed`, Math.floor(totalPlaytime))
 
-    if (runner === 'gog') {
-      await updateGOGPlaytime(appName, startPlayingDate, finishedPlayingDate)
+    const { disablePlaytimeSync } = GlobalConfig.get().getSettings()
+    if (!disablePlaytimeSync) {
+      if (runner === 'gog') {
```

Comment:
maybe we can move this if before checking if the sync is disabled? because you disable play sync this will show a message for games of every store and can be confusing (like... I finish playing an Epic game and I have a message saying it's skipped, but if I enable it then it doesn't post the info because it's only for gog)